packages/kbn-unified-field-list/src/components/field_list_filters/field_type_filter.tsx

```diff
@@ -149,7 +149,7 @@ export function FieldTypeFilter<T extends FieldListItem = DataViewField>({
   return (
     <EuiPopover
       id="unifiedFieldTypeFilter"
-      panelClassName="euiFilterGroup__popoverPanel"
+      panelProps={{ css: { width: euiTheme.base * 18 } }}
```

Comment:
In Discover, the filter button width and the filter item height look larger than it was in 8.9:

Before:
<img width="400" alt="Screenshot 2023-07-25 at 12 08 48" src="https://github.com/elastic/kibana/assets/1415710/dfd670c2-645a-4e44-81d8-2d7e6bcd2f9c">

Now:
<img width="400" alt="Screenshot 2023-07-25 at 12 08 58" src="https://github.com/elastic/kibana/assets/1415710/1ce23f6d-556f-4417-8abf-ba9ab9643352">

Reply:
Thanks @jughosta I'll look into that!

Reply:
The filter button width should be fixed as with https://github.com/elastic/kibana/pull/162209/commits/14e74692722afe0ebb981bf812e6f87a267fbaf4 - I hadn't realized folks in Kibana were using `EuiFilterButton` outside of `EuiFilterGroup`, so that's my bad :)

Regarding the filter item height - this looks like it was affected by https://github.com/elastic/kibana/pull/161592 (Sass styling order). I pushed up a specificity override for now (https://github.com/elastic/kibana/pull/162209/commits/f5a55e0c537bf72255c04dfc9bca563d123a4d45) which we can remove once more of EUI is on Emotion 🤞 

Thanks for your eagle eye on these things @jughosta, you're amazing!!

Reply:
Thanks for the fixes, @cee-chen! You are amazing! 😉